### PR TITLE
Create Openwpm.com.xml

### PR DESCRIPTION
**Pull request by ekaris**

### CODE REVIEW

**Comment by ekaris** [comment]
@J0WI Good to go

**Comment by J0WI** [comment]
Can you also comment that `(www.)openwpm.com` does not exist?

**Comment by ekaris** [comment]
Done

File: src/chrome/content/rules/Openwpm.com.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!--
 	Non working:
```
Comment (J0WI) [inline src/chrome/content/rules/Openwpm.com.xml]: They don't exist

**Comment by ekaris** [comment]
Good?

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**